### ISSUE TITLE
RISCV CPU supported?

### ISSUE THREAD

**Issue by HelloAllen8893** [intel/isa-l#239]
Hello, we are using a riscv CPU now, I have already built the isa-l code with cross compiler, but i am not sure whether this code has already support the riscv arch, any ideas?

**Comment by pablodelara** [comment]
No, most of the functions available in ISA-L are written in assembly for specific architectures (x86/ARM), and RISC-V is not one of them

**Comment by tipabu** [comment]
I see that the upcoming 2.32 release will [declare some level of support](https://github.com/intel/isa-l/commit/359e2ac1af26f4af0be490b1f295ef67d38a5de7), with more and more functionality [getting optimizations](https://github.com/intel/isa-l/commit/f74b0d27aba543657c17d9a3c225c61c6a8a846e). Is there a burn-down list of other places that should be optimized, or should we consider this resolved since we [added RISC-V CI](https://github.com/intel/isa-l/commit/91da2ada9af1ea63a024f235a7a6c2ff7f757b5e)?

Speaking of 2.32: is there an estimate for when the release will be finalized?
